src/backend/marsha/core/management/commands/clean_aws_elemental_stack.py

```diff
@@ -0,0 +1,90 @@
+"""Check every existing medialive channel and check if the related video is still usable."""
+from datetime import timedelta
+
+from django.conf import settings
+from django.core.management.base import BaseCommand
+from django.utils import timezone
+
+from marsha.core.defaults import DELETED, ENDED
+from marsha.core.models import Video
+from marsha.core.utils.medialive_utils import (
+    delete_aws_element_stack,
+    delete_mediapackage_channel,
+    list_medialive_channels,
+)
+from marsha.core.utils.time_utils import to_datetime
+
+
+def generate_expired_date():
+    """Generate a datetime object NB_DAYS_BEFORE_DELETING_AWS_ELEMENTAL_STACK days in the past."""
+    return timezone.now() - timedelta(
+        days=settings.NB_DAYS_BEFORE_DELETING_AWS_ELEMENTAL_STACK
+    )
+
+
+class Command(BaseCommand):
+    """
+    Once a live started, all AWS elemental stack are created. Once stopped, the
+    instructor must do an action. Restart it and/or convert it in VOD. If
+    nothing is done, the AWS element resources are leaved unused and use the quota
+    we have on our AWS account.
+    This resources unused must be removed after several days of inactivity and the video
+    move in a DELETED state.
+    """
+
+    help = (
+        "Once a live started, all AWS elemental stack are created. Once stopped, the "
+        "instructor must do an action. Restart it and/or convert it in VOD. If "
+        "nothing is done, the AWS element resources are leaved unused and use the quota "
+        "we have on our AWS account."
+        "This resources unused must be removed after several days of inactivity and the video "
+        "move in a DELETED state."
+    )
+
+    def handle(self, *args, **options):
+        """Execute management command."""
+        expired_date = generate_expired_date()
+        for medialive_channel in list_medialive_channels():
+            # Don't work with stack not belonging to the current environment
+            if (
+                medialive_channel.get("Tags", {}).get("environment")
+                != settings.AWS_BASE_NAME
+            ):
+                continue
+
+            # If the live is not idle, skip it
+            if medialive_channel.get("State") != "IDLE":
```

Comment:
I guess you did not use the `default.IDLE` value because it is not the same scope, right?

Reply:
Yes and AWS use their state in CAPITAL case, we don't. Or I can use `IDLE.upper()` maybe ?